features/sentinel/core/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -46,19 +45,4 @@
         </service-properties>
     </service>
 
-    <!-- REST Client used to communicate with the OpenNMS controller -->
-    <bean id="restClient" class="org.opennms.distributed.core.impl.ScvEnabledRestClientImpl">
```

Comment:
Should we remove the `RestClient` related classes as well?

Reply:
I was wondering that as well. I'd say yes to keep things clean. @cgorantla  what do you think?